src/main/drivers/light_ws2811strip_stdperiph.c

```diff
@@ -221,4 +221,12 @@ void ws2811LedStripDMAEnable(void)
     TIM_Cmd(timer, ENABLE);
     xDMA_Cmd(dmaRef, ENABLE);
 }
+
+#ifdef STM32F4
+bool ws2811LedStripDMA2Errata(void)
```

Comment:
'ws2811LedStripUsesDma2()` to make this a bit more indicative of what it returns?